CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 1.19.0 / 2025-08-19
+
+* [FEATURE] Configure gradle file to create multiple source sets for Compiler Plugin. See [#412](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/412)
+* [FEATURE] Move Kotlin Compiler logic into `Kotlin20` source sets. See [#413](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/413)
+* [FEATURE] Add Kotlin `2.2.0` support in Kotlin Compiler Plugin. See [#414](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/414)
+* [FEATURE] Add Kotlin `2.1.x` support in Kotlin Compiler Plugin. See [#415](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/415)
+* [FEATURE] Fix `IrFunctionExpressionImpl` in Kotlin `2.0.0 - 2.0.10`. See [#416](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/416)
+* [FEATURE] Add unit test for each Kotlin source set. See [#417](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/417)
+* [FEATURE] Upgrade Kotlin Version to `2.1.21` to remove metadata check error. See [#420](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/420)
+* [FEATURE] Merge `Feature/kotlin-source-sets` into `develop`. See [#421](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/421)
```

Comment:
this is rather `IMPROVEMENT` than `FEATURE` I think, since we didn't add anything new, we extended support.